README.md

```diff
@@ -1,44 +1,238 @@
 # Vault Integration Pack
+_StackStorm pack integration with HashiCorp Vault_
+
+*Author:* steve.neuharth <steve.neuharth@target.com>
+
+## Maintainers
+Active pack maintainers with review & write repository access and expertise with vault:
+* Jacob Floyd ([@cognifloyd](https://github.com/cognifloyd)) <cognifloyd@gmail.com> Copart
+* Carlos ([@nzlosh](https://github.com/nzlosh))
+
+### Contributors
+- Andy Moore <andrew.moore@pearson.com>
+- Jacob Floyd <cognifloyd@gmail.com>
+- Carlos <nzlosh@yahoo.com>
```

Comment:
```suggestion
```

I don't think it makes sense to mention every possible contributor. Maintainers feels like enough to me and follows the same format we're using in other packs.